corehq/messaging/tasks.py

```diff
@@ -101,6 +101,10 @@ def _get_cached_rule(domain, rule_id):
         return rules[0]
 
 
+def get_cached_rule(domain, rule_id):
+    return _get_cached_rule(domain, rule_id)
```

Comment:
how about editing the original function itself to be public?

Reply:
made this change. [49f9c15](https://github.com/dimagi/commcare-hq/pull/32671/commits/49f9c1580276337337574716a07ad0f5ed07240d)